pkg/builder/webhook.go

```diff
@@ -53,6 +55,15 @@ func (blder *WebhookBuilder) For(apiType runtime.Object) *WebhookBuilder {
 	return blder
 }
 
+// HandlerFor takes a admission.admissionFor interface.
+//
+// If the given object implements the admission.DefaulterFor interface, a MutatingWebhook will be wired for this type.
+// If the given object implements the admission.ValidatorFor interface, a ValidatingWebhook will be wired for this type.
+func (blder *WebhookBuilder) HandlerFor(forType admission.For) *WebhookBuilder {
```

Comment:
Would something like `WebhookFor` be better? I worry that `Handler` is such a generic term.

Reply:
Not sure if that's clear either, wouldn't the `WebhookBuilder` be in charge of returning the webhook?

Reply:
I tried to do that, but I couldn't get it to work with the DeepCopy and subsequent Decode. (because the webhook struct is not registered as type for the decoding). Do you have a link to an existing implementation?

Reply:
ah ok, I didn't test it I just assumed that it should work.

Reply:
Okay, I just thought maybe there is a way and I just couldn't find it :)